src/Query/ADQL_parser/ADQL_parser/init_column_reference.cxx

```diff
@@ -81,6 +81,17 @@ void ADQL_parser::init_column_reference() {
     qualifier_column %= (tap_upload | identifier) >> '.' >> identifier;
     qualifier_column.name("qualifier column");
     column_reference %= catalog_schema_qualifier_column | schema_qualifier_column |
-                        qualifier_column | identifier;
+                        qualifier_column | column_name;
```

Comment:
Wondering if 'column_name' being assigned with 'identifier' already?

Reply:
init_columns.cxx has this: `column_name %= identifier;`.

Reply:
init_columns() is called after init_column_reference(). Order does not matter?

Reply:
RIght, the order in which the init_XXX() functions are called in ADQL_parser.cxx does not matter.  I scrambled them completely and it had no effect on either the libadql test suite or the query_server-level regression tests.  But I've reverted the change for now and will perhaps do a larger-scale cleanup/reorganization of the initialization code later on.